src/roktManager.ts

```diff
@@ -126,6 +128,8 @@ export default class RoktManager {
         // Launcher options are set here for the kit to pick up and pass through
         // to the Rokt Launcher.
         this.launcherOptions = { sandbox, ...options?.launcherOptions };
+
+        this.domain = options?.domain;
```

Comment:
Our instructions usually reference domain to include the protocol, for example `const ROKT_DOMAIN = "https://apps.rokt-api.com";`. I think we should expect that a user might pass in a domain with and without a protocol, we should wrap some smarts around this to strip that out.

Reply:
In [the snippet](https://docs.rokt.com/developers/integration-guides/getting-started/ecommerce/ecommerce-sdk-integration), it actually says

```
mParticle.config.domain = ROKT_DOMAIN.split('//')[1]
```

to account for this